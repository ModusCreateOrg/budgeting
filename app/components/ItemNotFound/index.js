/**
 * Created by joaogabriellima on 16/11/17.
 */

import * as React from 'react';

const ItemNotFound = () => (
  <div>
    <h1>Item not found. </h1>
    <h2>Make sure the ID is correct and try again</h2>
  </div>
);

export default ItemNotFound;
