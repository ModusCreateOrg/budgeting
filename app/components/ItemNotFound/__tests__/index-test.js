import React from 'react';
import renderer from 'react-test-renderer';
import ItemNotFound from 'components/ItemNotFound';

it('renders correctly', () => {
  const tree = renderer.create(<ItemNotFound />).toJSON();
  expect(tree).toMatchSnapshot();
});
