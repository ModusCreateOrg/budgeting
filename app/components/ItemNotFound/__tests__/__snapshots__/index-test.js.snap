// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <h1>
    Item not found. 
  </h1>
  <h2>
    Make sure the ID is correct and try again
  </h2>
</div>
`;
